ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/management-api-http.yml

```diff
@@ -9,7 +9,105 @@ management_api_http:
       access_group: ACL-API
     - name: MGMT
       access_group: ACL-API
+  protocol_https_certificate:
+    certificate: '-----BEGIN CERTIFICATE-----
 
+      MIIDhzCCAm+gAwIBAgIQU7lKh63BnEKJepq2ZX0KmTANBgkqhkiG9w0BAQsFADBe
```

Comment:
Please use multiline yaml with pipe and avoid the extra newlines.

Reply:
We can replace the cert with some `<certstring>` placeholder

Reply:
Okay, shall we add <private_key> also?

Reply:
Fixed!